python/graphframes/connect/graphframes_client.py

```diff
@@ -117,6 +119,32 @@ def setIntermediateStorageLevel(self, storage_level: StorageLevel) -> Self:
         self._storage_level = storage_level
         return self
 
+    def requiredSrcColumns(self, colName: str, *colNames: str) -> Self:
```

Comment:
To be honest, I want to slowly deprecate/replace all the JVM-like camelCase methods in Python API by the pythonic snake_case. What to you think about change it to avoid generating more code that will be deprecated?